### ISSUE TITLE
knetattach files exist in the plasma-desktop package

### ISSUE THREAD

**Issue by fries1234** [kth5/archpower#62]
Looks like knetattach, a KDE plasma package, has files inside plasma-desktop so you get an error when trying to install knetattach.

```
error: failed to commit transaction (conflicting files)
knetattach: /usr/bin/knetattach exists in filesystem (owned by plasma-desktop)
knetattach: /usr/share/applications/org.kde.knetattach.desktop exists in filesystem (owned by plasma-desktop)
```

**Comment by kth5** [comment]
New plasma/kde is coming to `[testing]` towards the end of the day. I'll have a look at this.

**Comment by kth5** [comment]
Fixed by `plasma-desktop 5.27.8-1.1`
